content/01-etl/01-raw-data-refresh.qmd

```diff
@@ -0,0 +1,148 @@
+---
+title: "ETL Step 1 - Raw Data Refresh"
+date: "`r lubridate::now(tzone = 'EST')` EST"
+output: html_document
+---
+
+This notebook uses the [bikeHelpR](https://packagemanager.rstudio.com/client/#/repos/1/packages/bikeHelpR) package to update a database with the latest bike share data. The package pulls data from <https://capitalbikeshare.com> which provides an API to access bike share data. The raw data is written to the *Content DB* database to the *bike_raw_data* table.
+
+```{r setup, include=FALSE}
+knitr::opts_chunk$set(warning = FALSE, message = FALSE) 
+
+library(dplyr)
+library(odbc)
+
+# The following packages are not directly called, but are requirements. They
```

Comment:
I *think* these can probably go away now that I fixed the `bikeHelpR` dependencies. You'll have to reinstall and update the manifest.

Reply:
Updated, see 85cd664.